src/commands/login-new.ts

```diff
@@ -1,16 +1,18 @@
-const { cryptoRandomObjectId } = require('@apify/utilities');
-const { flags: flagsHelper } = require('@oclif/command');
-const computerName = require('computer-name');
-const cors = require('cors');
-const express = require('express');
-const inquirer = require('inquirer');
-const open = require('open');
-
-const { ApifyCommand } = require('../lib/apify_command');
-const outputs = require('../lib/outputs');
-const { useApifyIdentity } = require('../lib/telemetry');
-const { getLoggedClient } = require('../lib/utils');
-const { getLocalUserInfo } = require('../lib/utils');
+import { Server } from 'http';
```

Comment:
this command might be worth trying manually, as it does a bit more magic than the rest of them

Reply:
She works

![image](https://github.com/apify/apify-cli/assets/17960496/61e48f00-0316-42e0-9d98-80bc782301ae)